ops/framework.py

```diff
@@ -328,19 +329,24 @@ def __str__(self):
 
 class SQLiteStorage:
 
+    DB_LOCK_TIMEOUT = timedelta(hours=1).total_seconds()
```

Comment:
Sounds a bit better to keep that as a timedelta, and convert where necessary.

Reply:
https://github.com/canonical/operator/pull/141/commits/f6004043ef0519edfe7dee7aace2ccab22300d4a